.github/workflows/run-tests.yml

```diff
@@ -122,22 +122,19 @@ jobs:
                      --env PUSH_TO_CDASH=true \
                      --env WITH_3D=${WITH_3D} \
                      qgis/qgis3-build-deps:${DOCKER_TAG} \
-                     /root/QGIS/.docker/docker-qgis-build.sh
-          docker commit qgis_container qgis_image
+                     /bin/bash -c "/root/QGIS/.docker/docker-qgis-build.sh; ctest -V -S /root/QGIS/.ci/config_build.ctest"
```

Comment:
can we integrate the ctest line in the bash file?

Reply:
Done